### PR TITLE
⬆️ Upgrade Tailscale to 1.30.1

### PR DESCRIPTION
**Pull request by jeffothy**
# Proposed Changes

https://github.com/tailscale/tailscale/releases/tag/v1.28.0
https://github.com/tailscale/tailscale/releases/tag/v1.30.0
https://github.com/tailscale/tailscale/releases/tag/v1.30.1

## Related Issues

Supersedes https://github.com/hassio-addons/addon-tailscale/pull/106

### CODE REVIEW

File: tailscale/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN \
         ipcalc=1.0.1-r0 \
         iproute2=5.17.0-r0 \
         iptables=1.8.8-r1 \
-        nginx=1.22.0-r0 \
+        nginx=1.22.0-r1 \
```
Comment (frenck) [inline tailscale/Dockerfile:16]: This is not related to this PR, please revert it.

**Review by frenck — CHANGES_REQUESTED** [review]

File: tailscale/Dockerfile (hunk 1 above)
Comment (jeffothy) [reply tailscale/Dockerfile]: Done.

Comment (jeffothy) [reply tailscale/Dockerfile]: @frenck Are you going to fix this nginx dep bump in another PR then?

Builds fail without it due to:
```
ERROR: unable to select packages:
  nginx-1.22.0-r1:
    breaks: world[nginx=1.22.0-r0]
```

Comment (jeffothy) [reply tailscale/Dockerfile]: I have created https://github.com/hassio-addons/addon-tailscale/pull/109 for that purpose. Hopefully that helps!

Comment (frenck) [reply tailscale/Dockerfile]: > @frenck Are you going to fix this nginx dep bump in another PR then?
> 
> 
> 
> Builds fail without it due to:
> 
> ```
> 
> ERROR: unable to select packages:
> 
>   nginx-1.22.0-r1:
> 
>     breaks: world[nginx=1.22.0-r0]
> 
> ```

Sure, when I have time.

Comment (jeffothy) [reply tailscale/Dockerfile]: > Sure, when I have time.

No problem! Perhaps you missed my [other comment](https://github.com/hassio-addons/addon-tailscale/pull/108#discussion_r963022505) above. I tested https://github.com/hassio-addons/addon-tailscale/pull/109 build locally and it appears to be good to go.

Comment (frenck) [reply tailscale/Dockerfile]: I didn't mis it, I am abroad not able to work on things.

File: tailscale/Dockerfile [hunk 2]
```diff
@@ -25,7 +25,7 @@ RUN \
     && if [ "${BUILD_ARCH}" = "i386" ]; then ARCH="386"; fi \
     \
     && curl -L -s \
-        "https://pkgs.tailscale.com/stable/tailscale_1.26.0_${ARCH}.tgz" \
+        "https://pkgs.tailscale.com/stable/tailscale_1.30.0_${ARCH}.tgz" \
```
Comment (frenck) [inline tailscale/Dockerfile:28]:
```suggestion
        "https://pkgs.tailscale.com/stable/tailscale_1.30.1_${ARCH}.tgz" \
```

**Review by frenck — APPROVED** [review]
Thanks @jeffothy 👍
